types/aws-cloudfront-function/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for non-npm package AWS CloudFront Functions 1.0
+// Project: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/functions-event-structure.html
+// Definitions by: Jakub Zelenka <https://github.com/bukka>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
```suggestion
```
This version is already phased out here on DT, please left blank and add minimum only if required by TS feature used

Reply:
Makes sense. Although it's present in other packages but will remove in another PR so I don't bother the maintainers with a new commit just for this. Doesn't seem like a big issue though.